rotkehlchen/chain/ethereum/modules/liquity/trove.py

```diff
@@ -627,7 +627,9 @@ def get_history_events(
             changes = self._process_trove_events(trove['changes'], from_timestamp, to_timestamp)
             result.append(changes)
         # Flatten the result (list of lists to list)
-        return reduce(add, result)
+        if len(result) == 0:
```

Comment:
Sorry it was my fault at the moment of writing it. `if result` is equivalent to `len(result) != 0`